packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -311,7 +311,7 @@ const Cart = () => {
         return <CartSkeleton />
     }
 
-    if (!isLoading && !isProductsLoading && !basket?.productItems?.length) {
+    if (!isLoading && !basket?.productItems?.length) {
```

Comment:
If we don't have products in the basket, then the products query never fires, so we don't need to check for it here.